modules/redpanda/mounts/bootstrap.yaml.tpl

```
@@ -22,3 +22,10 @@ data_transforms_enabled: true
 {{- if .AutoCreateTopics }}
 auto_create_topics_enabled: true
 {{- end }}
+
+{{- if .ExtraBootstrapConfig }}
+{{- range $key, $value := .ExtraBootstrapConfig }}
+{{ $key }}: {{ $value }}
+{{- end }}
+{{- end }}
```

suggestion: I don't believe you need the if here as zero iterations should be fine?
```suggestion
{{- range $key, $value := .ExtraBootstrapConfig }}
{{ $key }}: {{ $value }}
{{- end }}
```